docs/source/tusupload.rst

```diff
@@ -1,88 +1,99 @@
 TUS resumable upload
 ====================
 
-Plone REST API supports the TUS Open Protocol for Resumable File Uploads.
-See http://tus.io/ for more information on TUS.
+plone.restapi supports the `TUS Open Protocol <http://tus.io>`_ for resumable file uploads.
+The two TUS endpoints to upload '@tus-upload' a file and to replace an existing file '@tus-replace' are available on all 'folderish resources' in Plone (content types that implement the IFolderish interface, e.g. a 'Folder').
```

Comment:
@tisto In fact it's more complicated. `POST @tus-upload` is limited to folderish resources, but `PATCH @tus-upload` works on IContentish resources. That's because of the replace functionality. Replace works on any content object while creating new files works only on folderish objects only.
While looking again at the ZCML I found some minor issues. `HEAD @tus-upload` should be registered for IContentish and `PATCH @tus-upload` should have a conditional permission check (cmf.AddPortalContent if creating a new file, cmf.ModifyPortalContent if replacing an existing file). I can fix that tomorrow.